### DISCUSSION TITLE
Error loading large images

### DISCUSSION THREAD

**Discussion by Urdabaev99** [mistic100/Photo-Sphere-Viewer#966] [General]
I use your library i put 360 degree pictures there but when i upload a 20-30mb image then the following error appears in the browser "Uncaught (in promise) TypeError: Failed to execute 'enqueue' on 'ReadableStreamDefaultController': Cannot enqueue a chunk into a closed readable stream
   at three.min.js:7:519065" how do I solve this problem?

**Comment by mistic100** [comment]
Please use issues for support request.

You will have to provide a demo of your viewer where the bug is encountered (without using minified files because it is impossible to debug).
